src/autocert/example_test.go

```diff
@@ -5,12 +5,11 @@
 package autocert_test
 
 import (
-	"crypto/tls"
 	"fmt"
 	"log"
 	"net/http"
 
-	"github.com/futurice/alley-oop/src/autocert"
```

Comment:
what? Should this not be pointing to our own source code?

Reply:
Maybe they should. I updated this and the other occurence to point to this repo.